vizro-core/src/vizro/models/_components/form/slider.py

```diff
@@ -112,7 +112,7 @@ def build(self):
                             persistence_type="session",
                             className="slider_input_field_right" if self.step else "slider_input_field_no_space_right",
                         ),
-                        dcc.Store(id=f"{self.id}_temp_store", storage_type="session"),
+                        dcc.Store(id=f"{self.id}_input_store", storage_type="session", data=self.value or self.min),
```

Comment:
Based on the `RangeSlider.build` implementation, perhaps it makes sense to calculate variable `init_value = self.value or self.min` at the beginning of the `Slider.build` method and then just reuse it in three places below.